### PR TITLE
fix(VSelect): add watcher on items for hide-no-data

### PR DESCRIPTION
**Pull request by erenertas123**
Hi

I dont know if anyone working on this but i try to resolve issue number #18556 .

When items added on focus function menuDisabled couldn't be updated cause its run after onMousedownControl so i added watch on items when items updated and input is focused menu will open on with or without hide-no-data attribute.

### CODE REVIEW

**Comment by erenertas123** [comment]
Hello,

I also try to resolve issue number #18665 in the same fork i dont know if it is proper way to do it. Should i open new pr from another fork?

File: packages/vuetify/src/components/VSelect/VSelect.tsx [hunk 1]
```diff
@@ -273,6 +273,12 @@ export const VSelect = genericComponent<new <
         vTextFieldRef.value.value = ''
       }
     }
+    
+    watch(items, () => {
```
Comment (johnleider) [inline packages/vuetify/src/components/VSelect/VSelect.tsx:277]: This seems ripe for issues with dynamic items.

**Comment by johnleider** [comment]
Can you remove the unrelated changes to stepper?

**Comment by johnleider** [comment]
I opted to fix this due to time constraints. Fix is here 48233095aa6b379655eb45e1847fa6996cc7ef8f

Thank you for your contribution and interest in improving Vuetify! Make sure to join us in the [Discord community](https://community.vuetifyjs.com).

**Pull request closed without merging**